appcues/src/main/java/com/appcues/debugger/screencapture/AndroidTargetingStrategy.kt

```diff
@@ -115,7 +115,7 @@ private fun View.asCaptureView(screenBounds: Rect): ViewElement? {
         getGlobalVisibleRect(globalVisibleRect).not() ||
         // if the view is not currently in the screenshot image (scrolled away), ignore
         // (this is possibly a redundant check to item above, but keeping for now)
-        Rect.intersects(globalVisibleRect, screenBounds).not()
+        screenBounds.contains(globalVisibleRect).not()
```

Comment:
can you expand on this change?

Reply:
I think it is probably not changing anything, in reality. I was trying to change the logic here to say: the screen bounds has to contain the entire view rect -- but this `globalVisibleRect` is just the visible rect - the part that is in the screen bounds, so this is always the same result as checking if it intersects -- and in reality, this is probably all redundant since we now check the return value on `getGlobalVisibleRect` and that indicates something is not on the screen (if false)